src/lib/LED/devLED.cpp

```diff
@@ -296,6 +296,10 @@ static int event()
             // technically nocrossfire is {10,100} but {20,100} is close enough
             return flashLED(GPIO_PIN_LED_RED, GPIO_LED_RED_INVERTED, LEDSEQ_RADIO_FAILED, sizeof(LEDSEQ_RADIO_FAILED));
         }
+        else if (GPIO_PIN_LED != UNDEF_PIN)
+        {
+            return flashLED(GPIO_PIN_LED, GPIO_LED_RED_INVERTED, LEDSEQ_RADIO_FAILED, sizeof(LEDSEQ_RADIO_FAILED));
+        }
```

Comment:
Since this is now going to be used on the RX we should probably have a different blink-code for it. Making it the same as "Radio failed" seems like a bad idea.

Reply:
What if I told you it was the "No crossfire" blink code instead? 😄 I didn't really think even needed a blink pattern, so I could remove this and the LED will just be on solid, is that better? This state is the "locked up" state that only exists from the time a loanee with a receiver that is in "Returnable" mode has "Returned Model". When the receiver is rebooted it does normal stuff, so this is only used during that short window to let the user know it is not going to accept a connection.

The No Crossfire / Radio Failed might slightly perplex someone who has turned this on because they don't know what it does and somehow managed to get to this point and can see their RX LED, but the first thing everyone would try (reboot the RX) returns it to normal operation so do you think this is something we should create a new state for and invent a new blink code that is distinguishable for this very unique mode that's developed exclusively for a technical ops team?

I can I just feel like blink patterns for things users might actually encounter need to be distinguishable and every time we make one it gets harder to tell them apart.

Reply:
Maybe just turn the LED off completely to show that it's shut down.